qualysapi/config.py

```diff
@@ -29,10 +30,10 @@ class QualysConnectConfig:
     from an ini file.
     """
 
-    def __init__(self, filename=qcs.default_filename, remember_me=False, remember_me_always=False):
+    def __init__(self, filename=qcs.default_filename, section='info', remember_me=False, remember_me_always=False):
```

Comment:
This "section" part is the one that I added, as we were having the issue were a user couldn't user both Qualys WAS and Qualys VM, and the reason of it is that your fork was looking only into the [qualys] section of the config file, which was hardcoded into qualysapi.